frontend/src/assets/css/header.scss

```diff
@@ -0,0 +1,31 @@
+$navbar-bg: #f8f9fa;
+$navbar-shadow: 0 2px 4px #0000001a;
+$navbar-hover-bg: #e2e3e5;
+
+.navbar {
+  margin: 10px;
+  padding: 15px 0;
+  background-color: $navbar-bg;
+  border-radius: 5px;
+  box-shadow: $navbar-shadow;
+}
+
+.navbar-content {
+  display: flex;
+  justify-content: space-between;
+  padding-right: 10%;
+  padding-left: 10px;
+}
+
+.nav-link {
+  padding: 5px 10px;
+  color: dimgray;
```

Comment:
use color from design

Reply:
I use this tool to pick the color: 
<img width="960" alt="Captura de pantalla 2023-08-19 222355" src="https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/assets/90299049/6f6e86c5-e8c9-4462-b5a0-8c0350cb4155">

Reply:
it's better to use figma colors, not side color picker. To do this you need to select the element and you will see it's color in the side panel
<img width="680" alt="Screenshot 2023-08-20 at 19 21 56" src="https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/assets/64521658/da733a78-a595-412b-b6f0-a3c80e73b5d1">